lib/n3h-mod-connection-wss/wss-connection.js

```diff
@@ -55,7 +55,14 @@ class ConnectionBackendWss extends AsyncClass {
           ws.close(1001, 'stale connection')
           await this._handleClose(id)
         } else if (Date.now() - ws._$_lastMsg >= 2000) {
-          ws.ping()
+          try {
+            ws.ping()
```

Comment:
@lucksus - it looks like this `ping()` and the `send()` below use callbacks for asynchronous behavior (https://github.com/websockets/ws/blob/HEAD/doc/ws.md#websocketpingdata-mask-callback), so the try/catches may not catch errors.

promisify / await:
```javascript
const { $p } = require('../n3h-common')
await $p(cb => { ws.ping(cb) })
```

Reply:
I just followed the stacktrace that was printed out to these lines. Adding this try catch fixes the problem.